src/storage/conversion/QuadToRdfConverter.ts

```diff
@@ -30,6 +31,18 @@ export class QuadToRdfConverter extends BaseTypedRepresentationConverter {
     const contentType = getConversionTarget(await this.getOutputTypes(INTERNAL_QUADS), preferences.type)!;
     let data: Readable;
 
+    // Remove the ResponseMetadata graph as we never want to see it in a serialization
```

Comment:
Add some more comments explaining that we want to find a better solution than this.